pkg/util/shell/shell.go

```diff
@@ -74,7 +74,7 @@ func GetPrefixSuffixDelimiterForSet(userShell string, pathVar bool) (prefix, suf
 	switch userShell {
 	case "fish":
 		prefix = "set -gx "
-		suffix = "\";\n"
+		suffix = ";\n"
```

Comment:
@quagliero , please updated this section as mentioned here https://github.com/minishift/minishift/issues/1006#issuecomment-307301006